templates/webapi/test/settings.html.ep

```diff
@@ -4,9 +4,22 @@
     </thead>
     <tbody>
         % my $s = $job->settings_hash;
+        % my @link_keys = split ",", app->config->{job_settings_ui}->{keys_to_render_as_links};
+        % my $links = join '|', @link_keys;
         % for my $k (sort keys %$s) {
             % my $v = $s->{$k};
-            <tr><td><%= $k %></td><td><%= $v =~ m{^https?://} ? link_to $v => $v : $v %></td></tr>
+            <tr><td><%= $k %></td>
+              % if ($v =~ m{^https?://|$links}) {
```

Comment:
These checks are still only done on the values and not on the keys as I suggested. You weirdly use `keys_to_render_as_links` but then although the code doesn't cares about the settings keys at all and applies the configuration to values.

Not sure whether we actually reached a consensus how this feature should work but in my opinion it should be either:

* https://github.com/os-autoinst/openQA/pull/3203/files#r452719772
* or https://github.com/os-autoinst/openQA/pull/3203/files#r452728210 (option 4)